### ISSUE TITLE
`pulumi convert` should emit Python projects with our preferred defaults

### ISSUE THREAD

**Issue by joeduffy** [pulumi/pulumi#9955]
Currently `pulumi convert` emits a project file with just `runtime: python` in it:

```
runtime: python
```

Yet for most new projects we recommend using the automatic `virtualenv` defaults:

```
runtime:
  name: python
  options:
    virtualenv: venv
```

That yields this error if you naively try to deploy the result of `pulumi convert`:

```
Diagnostics:
  pulumi:pulumi:Stack (pulumi-ansible-wordpress-py-dev):
    Traceback (most recent call last):
      File "/usr/local/bin/pulumi-language-python-exec", line 14, in <module>
        import pulumi
    ModuleNotFoundError: No module named 'pulumi'
    It looks like the Pulumi SDK has not been installed. Have you run pip install?
    If you are running in a virtualenv, you must run pip install -r requirements.txt from inside the virtualenv.

    error: an unhandled error occurred: Program exited with non-zero exit code: 1
```

Should we be emitting these project files with the preferred new defaults? Also, is this a problem for our other `x2pulumi` conversion tools? If yes, it may be worth a broader audit/fix of them.

As an aside, should we detect this error automatically, and suggest users opt into the auto-`virtualenv` feature? It's not clear how as an end user who stumbled upon this error, and a project that didn't use the new settings, would discover / know how to use it.

**Comment by Frassle** [comment]
> Also, is this a problem for our other x2pulumi conversion tools? If yes, it may be worth a broader audit/fix of them.

Yes probably, this is special cased to be added in `pulumi new` when making a new folder from a template.
